src/sentry/integrations/mixins/issues.py

```diff
@@ -99,17 +100,26 @@ def get_performance_issue_description_data(self, event):
         an integration's ticket description. Each integration will need to take
         this data and format it appropriately.
         """
-        spans, matched_problem = get_span_and_problem(event)
-        if not matched_problem:
-            return ""
+        if isinstance(event, GroupEvent) and event.occurrence is not None:
+            transaction_name = event.occurrence.evidence_data.get("transaction_name", "")
+            parent_span = event.occurrence.evidence_data.get("parent_span", "")
+            repeating_spans = event.occurrence.evidence_data.get("repeating_spans", "")
+            num_repeating_spans = event.occurrence.evidence_data.get("num_repeating_spans", 0)
 
-        parent_span, repeating_spans = get_parent_and_repeating_spans(spans, matched_problem)
-        transaction_name = get_span_evidence_value_problem(matched_problem)
-        parent_span = get_span_evidence_value(parent_span)
-        repeating_spans = get_span_evidence_value(repeating_spans)
-        num_repeating_spans = (
-            str(len(matched_problem.offender_span_ids)) if matched_problem.offender_span_ids else ""
-        )
+        else:
```

Comment:
Will this stuff be removed later on? When do we expect it to get here?

Reply:
once the migration is done and GA'ed I will remove this since all perf issue events will be GroupEvents